VirtusLab-Open-Source/strapi-plugin-comments
# Added Polish Language

## Ticket

Non

## Summary

What does this PR do/solve?

Added Polish Language

## Test Plan

1. Change locale on admin/app.ts and rebuild application
2. Change strapi language on login page to "Polish"
3. Verify translation on plugin page
